src/hotspot/share/runtime/semaphore.hpp

```diff
@@ -50,7 +50,12 @@ class Semaphore : public CHeapObj<mtSynchronizer> {
   Semaphore(uint value = 0) : _impl(value) {}
   ~Semaphore() {}
 
-  void signal(uint count = 1) { _impl.signal(count); }
+  void signal(uint count = 1) { _impl.signal(count, false /* ignore_overflow */); }
```

Comment:
Please change comment to:

`/* don't ignore overflow */`

as that describes what the `false` argument actually means, rather than just showing what parameter it is providing the value for.